docs/data/sdks/typescript-node/ampli.md

```diff
@@ -197,13 +197,13 @@ Call `setGroup()` to associate a user with their group (for example, their depar
 === "TypeScript"
 
     ```typescript
-    ampli.setGroup('userId', groupType', 'groupName');
+    ampli.client.setGroup('userId', 'groupType', 'groupName');
```

Comment:
Do we need to update other runtimes as well? This change is for all platforms

Reply:
Checking.

Reply:
We do need to update others. Will did other changes in this pr.